app/Services/Campaign/ExportService.php

```diff
@@ -46,13 +48,28 @@ public function queue(): self
         $this->campaign->export_date = date('Y-m-d');
         $this->campaign->saveQuietly();
 
-        Export::dispatch($this->campaign, $this->user, false);
-        Export::dispatch($this->campaign, $this->user, true);
+        $entitiesExport = CampaignExport::create([
```

Comment:
I feel like these should be created in the Job rather than here and passed as a parameter